packages/replay-orchestrator/src/test-run/parallel-tests/execute-test-run.ts

```diff
@@ -259,6 +259,7 @@ export const executeTestRun = async ({
             testRunId: testRun.id,
             replayEventsDependencies,
             cookiesFile: null,
+            logLevel: logger.getLevel(),
```

Comment:
Note: this is duplicated, but I'll refactor in main repo. I'm going to delete this package in next PR